### ISSUE TITLE
After giving close-session RPC showing as Already connected when trying to connect again.

### ISSUE THREAD

**Issue by sowmi18** [CESNET/netopeer2#1244]
Hi @michalvasko,

I connected to a netconf session and when i gave close-session RPC, i got (netconf-session-end) in the server side. But when i try to connect to the server again it shows ** Already connected to 192.168.1.10]**.
If i give disconnect command, i can able to connect again without any problem.
Also what is the difference between close-session RPC and disconnect cli command.

```
connect --ssh --host 192.168.1.10 --login operator

Interactive SSH Authentication
Type your password:
Password: 
> subscribe
OK
> user-rpc
<close-session` xmlns="urn:ietf:params:xml:ns:netconf:base:1.0"></close-session>

OK
> 
nc ERROR: SSH channel unexpected EOF.
> 
> connect --ssh --host 192.168.1.10 --login operator
cmd_connect: Already connected to 192.168.1.10.`
```

**Comment by michalvasko** [comment]
Of course, if you are sending `close-session` by the `user-rpc` command, the CLI will not detect it and think that you are still connected. The only difference to using `disconnect` is that this command is known to CLI and it knows it terminates the NETCONF session so just use that. The CLI is far from perfect and is generally meant for testing purposes.

**Comment by sowmi18** [comment]
got it.Thank you for clarifying!

**Comment by sowmi18** [comment]
Hi @michalvasko,

If i give subscribe command in CLI, notifications are coming. But if i give the RPC for subscribe, notifications are not coming. Is this the expected one? Like the issue i already asked above.
1. using subscribe CLI command, getting notification
```
> subscribe --filter-xpath /o-ran-supervision:*
OK
notification (2027-04-14T04:09:52Z)
<supervision-notification xmlns="urn:o-ran:supervision:1.0"/>
```
2. Using subscribe RPC not getting notification
```
<create-subscription xmlns="urn:ietf:params:xml:ns:netconf:notification:1.0">
    <filter type="subtree">
            <supervision-notification xmlns="urn:o-ran:supervision:1.0"/>
    </filter>
</create-subscription>
```

**Comment by michalvasko** [comment]
> Is this the expected one?

Yes, this is expected. The RPC itself only tells the server to send notifications but the client must be able to asynchronously read them. The CLI is not able to recognize this from your custom RPC so the notifications are coming but the CLI is not reading and printing them.

**Comment by sowmi18** [comment]
okay. Is there any way to get the notification in the client side with the Custom RPC?

**Comment by michalvasko** [comment]
Naturally, in a generic libnetconf2-based NETCONF client. But not in *netopeer2-cli*.
